File: android/Demo/src/main/java/com/example/signdemo1/ChooseActivity.java
package com.example.signdemo1;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.Nullable;

public class ChooseActivity extends Activity {
  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    findViewById(R.id.sign_in_activity)
        .setOnClickListener(
            v -> startActivity(new Intent(ChooseActivity.this, SignInActivity.class)));
    findViewById(R.id.token_activity)
        .setOnClickListener(
            v -> startActivity(new Intent(ChooseActivity.this, IdTokenActivity.class)));
  }
}

File: android/Demo/src/main/java/com/example/signdemo1/ChooseActivity.kt
package com.example.signdemo1

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class ChooseActivity : AppCompatActivity() {

    private val TAG = "MainActivity"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<View>(R.id.sign_in_activity)?.setOnClickListener {
            startActivity(Intent(this, SignInActivity::class.java))
        }
        findViewById<View>(R.id.token_activity)?.setOnClickListener {
            startActivity(Intent(this, IdTokenActivity::class.java))
        }
    }

}